clang/test/dpct/usm-syclcompat.cu

```diff
@@ -0,0 +1,1090 @@
+// FIXME
```

Comment:
why FIXME in Line1?

Reply:
Copy from the original lit test. Will remove it.